### PR TITLE
Add r-roi.plugin.quadprog

### PR DESCRIPTION
**Pull request by janjagusch**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by janjagusch** [comment]
Rendered with https://github.com/bgruening/conda_r_skeleton_helper/commit/0c460dedff2e96548d526c06f1fe466769463160.

**Comment by janjagusch** [comment]
@conda-forge/r, please review. The Windows build is currently failing. Happy to skip that one, but I forgot I'll let you take a closer look first.

File: recipes/r-roi.plugin.quadprog/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    - r-slam
+
+test:
+  commands:
+    - $R -e "library('ROI.plugin.quadprog')"           # [not win]
+    - "\"%R%\" -e \"library('ROI.plugin.quadprog')\""  # [win]
+
+about:
+  home: http://roi.r-forge.r-project.org/, https://r-forge.r-project.org/projects/roi/
+  license: GPL-3.0-or-later
```
Comment (mfansler) [inline recipes/r-roi.plugin.quadprog/meta.yaml:49]: Only see `GPL-3`, so

```suggestion
  license: GPL-3.0-only
```

**Review by mfansler — CHANGES_REQUESTED** [review]
Good, just needs license fixed.

**Comment by janjagusch** [comment]
@mfansler, any clue why the `win-64` build is failing? Happy to skip the Windows build otherwise.

Same applies to https://github.com/conda-forge/staged-recipes/pull/27309 and https://github.com/conda-forge/staged-recipes/pull/27308#pullrequestreview-2282744327.

```
Encountered problems while solving:
  - package m2-conda-epoch-20230914-0_x86_64 has constraint msys2-conda-epoch <0.0a0 conflicting with msys2-conda-epoch-20160418-1
  - package m2w64-bwidget-1.9.10-2 requires msys2-conda-epoch 20160418, but none of the providers can be installed
  - package m2w64-bzip2-1.0.6-6 requires msys2-conda-epoch 20160418, but none of the providers can be installed
  - package m2w64-expat-2.1.1-2 requires msys2-conda-epoch 20160418, but none of the providers can be installed
  - package m2w64-fftw-3.3.4-6 requires msys2-conda-epoch 20160418, but none of the providers can be installed
```

**Review by mfansler — APPROVED** [review]
Thanks for adding! ❤️

**Comment by mfansler** [comment]
Hmm. This got messy because you originally had this in the commit history of `r-symphony` that I just merged.

**Comment by mfansler** [comment]
You'll need to add back the build scripts.

**Comment by mfansler** [comment]
> @mfansler, any clue why the `win-64` build is failing? Happy to skip the Windows build otherwise.
> 
> Same applies to #27308 and #27306.
> 
> ```
> Encountered problems while solving:
>   - package m2-conda-epoch-20230914-0_x86_64 has constraint msys2-conda-epoch <0.0a0 conflicting with msys2-conda-epoch-20160418-1
>   - package m2w64-bwidget-1.9.10-2 requires msys2-conda-epoch 20160418, but none of the providers can be installed
>   - package m2w64-bzip2-1.0.6-6 requires msys2-conda-epoch 20160418, but none of the providers can be installed
>   - package m2w64-expat-2.1.1-2 requires msys2-conda-epoch 20160418, but none of the providers can be installed
>   - package m2w64-fftw-3.3.4-6 requires msys2-conda-epoch 20160418, but none of the providers can be installed
> ```

Fine just to leave it, even with no explicit skip. It is [a known weird issue](https://github.com/conda-forge/staged-recipes/issues/25319) that seems to stem from underspecification in Windows solves when there is nothing forcing an epoch in the `build` environment, which happens with **noarch**. It is totally fine to ignore Windows failures for **noarch** recipes.

**Review by mfansler — APPROVED** [review]

**Pull request merged**